gogh.sh

```diff
@@ -132,6 +132,7 @@ declare -a THEMES=(
   'one-dark.sh'
   'one-half-black.sh'
   'one-light.sh'
+  'palenight'
```

Comment:
Hi :slightly_smiling_face: 
I feel like here should be `palenight.sh` instead of just `palenight`